node/pkg/fetcher/app.go

```diff
@@ -156,6 +158,8 @@ func (a *App) startAll(ctx context.Context) error {
 		return err
 	}
 
+	go a.WFetcher.Start(ctx)
```

Comment:
@nick-bisonai For now, we just plan to run it concurrently and later removed old fetchers, is that correct?

Reply:
old fetcher will still be required for dex fetchers but mostly yes

Reply:
are we planning to migrate the dex fetchers to ws as well?

Reply:
we can, haven't been thinking about that yet 🥹🥹